File: My-Blog-ServeOpen/appone/models/ip.py
from django.db import models

class Ip(models.Model):
    user_id = models.IntegerField(default=0, verbose_name='用户id')
    ip = models.CharField(max_length=20, null=True, blank=True, verbose_name='IP')
    province = models.CharField(max_length=20, null=True, blank=True, verbose_name='省')
    city = models.CharField(max_length=20, null=True, blank=True, verbose_name='市')
    create_time = models.DateTimeField(verbose_name='修改时间now')
    k_time = models.DateTimeField(verbose_name='修改时间', null=True, blank=True)
    mcount = models.IntegerField(default=0, verbose_name='某时间段内登录次数')
    dcount = models.IntegerField(default=0, verbose_name='登录天数')

    class Meta:
        db_table = 'ip'
        verbose_name = 'IP地址表'
        verbose_name_plural = verbose_name

    def __str__(self):
        return str(self.user_id) + '-' + str(self.ip)

File: My-Blog-Go/models/ip.go
package models

import (
	"time"
)

// IP IP地址表
type IP struct {
	ID         uint       `gorm:"primaryKey;column:id" json:"id"`
	UserID     int        `gorm:"column:user_id;default:0" json:"userId"`
	IP         *string    `gorm:"column:ip;size:20" json:"ip,omitempty"`
	Province   *string    `gorm:"column:province;size:20" json:"province,omitempty"`
	City       *string    `gorm:"column:city;size:20" json:"city,omitempty"`
	CreateTime time.Time  `gorm:"column:create_time" json:"createTime"`
	KTime      *time.Time `gorm:"column:k_time" json:"kTime,omitempty"`
	MCount     int        `gorm:"column:mcount;default:0" json:"mCount"`
	DCount     int        `gorm:"column:dcount;default:0" json:"dCount"`
}

func (IP) TableName() string {
	return "ip"
}
